helm/cloud-provider-vsphere/templates/daemonset.yaml

```diff
@@ -68,6 +64,7 @@ spec:
       serviceAccountName: {{ .Values.serviceAccount.name }}
       hostNetwork: true
       dnsPolicy: {{ .Values.daemonset.dnsPolicy }}
+      priorityClassName: system-node-critical
```

Comment:
Should we align with [CAPVCD ](https://github.com/giantswarm/cluster-api-provider-cloud-director-app/blob/a86279ee0b096ead6e87e8231e8e1ffba73dbf58/config/kustomize/patches/deployment-resources.yaml#L14)chart? 
(This is a question)